packages/lime-system/files/etc/config/lime-defaults

```diff
@@ -17,6 +17,7 @@ config lime network
 	option anygw_dhcp_limit '0'
 	option main_ipv6_address '2a00:1508:0a%N1:%N200::/64'
 	option bmx6_mtu '1500'
+	option bmx7_mtu '1500'
```

Comment:
If 1500 is fallback there is no need to put it in lime-default but in lime-examples right?

Reply:
No. By design all available options must exist in lime-defaults. In addition the MTU is not fallback, it is a needed parameter.